app/femr/ui/models/manager/IndexViewModelGet.java

```diff
@@ -33,5 +45,108 @@ public String getToday()
         String currentDate= dateUtils.getFriendlyDate(todayDate);
         return currentDate;
     }
+    public DateTime getTodayDate()
+    {
+        return  DateTime.now();
+
+    }
+    // Sets patient encounter items
+    public void setPatientEncounter(List<PatientEncounterItem> patientEncounter) {
+        this.encounterItem = patientEncounter;
+    }
+    // gets array list item of Patient Encounter Items
 
+    public PatientEncounterItem getEncounter (int i){
+        return encounterItem.get(i);
+    }
+
+    // gets array list of Patient Encounter Items
+    public List<PatientEncounterItem> getEncounter() {
+        return encounterItem;
+    }
+
+//returns turn around time in the form of a String
+    public String turnAroundTime(PatientEncounterItem item){
```

Comment:
A few suggestions for this viewmodel that you build - would like to hear your thoughts as well. Viewmodels are supposed to be "dumb" in that they don't do much logic. This `turnAroundTime()` method may come in useful elsewhere. i would consider moving the logic to our utilities:

https://github.com/FEMR/femr/tree/master/app/femr/util/calculations

and invoking it when needed (should be in the controller)